ShellCheck/Formatter/TTY.hs

```diff
@@ -44,13 +44,13 @@ colorForLevel level =
         "message" -> 1 -- bold
         "source"  -> 0 -- none
         otherwise -> 0 -- none
-    
+
 outputError file error = do
-    color <- getColorFunc
-    hPutStrLn stderr $ color "error" $ file ++ ": " ++ error
+    color <- getColorFunc $ ColorAuto -- FIXME: should respect --color
+    hPutStrLn stderr $ color "error ZZZ" $ file ++ ": " ++ error
```

Comment:
Oh, oops! Don't worry about it, I fixed it in b439041 by adding some formatting options instead of threading it through the result. 

Thanks for the fix!